install/merger/merger.go

```diff
@@ -0,0 +1,33 @@
+package merger
+
+const (
+	OnConflictLabel   string = "on-conflict"
+	ReplaceOnConflict string = "replace"
+)
+
+type Data interface {
```

Comment:
This is purely design related remark. This is used in one of the following ways:
- User calls Update
- User calls Merge and then Update

Usually when I see that the interface contains a methods that must be called is a specific order I try to understand if the interface could be redesigned.

We could have the following:
```
type Data interface {
  Labels() *map[string]string
  Override() error // makes Update 
  Merge() error // makes merge and then Update
}
```

The implementation would call either `Override` or `Merge`.  You can ignore it. I still think that this abstraction is not suitable here.

Reply:
Such approach would force on us further interface changes if modifying merger logic. I see interfaces as a separation between implementation and its usages. If add Merger/Override in the interface we are effectively breaking encapsulation because now is the external components (not the merger) that decide when and how to do merger's logic.

Reply:
I don't get your point. Creating interfaces with some hidden assumption about  method calling order is not a good practice IMHO.

Reply:
Interfaces removed.